### PR TITLE
MIPS -> PIPS

### PR DESCRIPTION
**Pull request by vkomenda**

### CODE REVIEW

File: polymesh_schema.json [hunk 1]
```diff
@@ -281,13 +281,13 @@
             "Community"
         ]
     },
-    "MIP": {
-        "index": "MipsIndex",
+    "PIP": {
+        "index": "PipsIndex",
```
Comment (fmiguelgarcia) [inline polymesh_schema.json:285]: This field has been renamed to   `id`, and I've just realized I forget to rename it in our schema

File: polymesh_schema.json [hunk 2]
```diff
@@ -281,13 +281,13 @@
...
         "proposal":"Call",
         "state":"ProposalState"
     },
     "Referendum": {
-        "index": "MipsIndex",
+        "index": "PipsIndex",
```
Comment (fmiguelgarcia) [inline polymesh_schema.json:290]: ditto.

File: polymesh_schema.json [hunk 3]
```diff
@@ -232,13 +232,13 @@
         "motions": "Vec<Motion>"
     },
     "Url": "Text",
-    "MipDescription": "Text",
-    "MipsMetadata": {
+    "PipDescription": "Text",
+    "PipsMetadata": {
         "proposer": "AccountKey",
         "index": "u32",
```
Comment (fmiguelgarcia) [inline polymesh_schema.json:238]: See comment below

Comment (vkomenda) [reply polymesh_schema.json:238]: Sorry, @fmiguelgarcia , please explain.

Comment (vkomenda) [reply polymesh_schema.json]: Got it.

**Pull request merged**